src/main/java/roomescape/exception/ReservationExceptionHandler.java

```diff
@@ -0,0 +1,15 @@
+package roomescape.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+@RestControllerAdvice
+public class ReservationExceptionHandler {
+
+    @ExceptionHandler
+    protected ResponseEntity<String> handleIllegalArgumentException(IllegalArgumentException e) {
+        return new ResponseEntity<>(e.getMessage(), HttpStatus.BAD_REQUEST);
```

Comment:
```suggestion
        return ResponseEntity.badRequest().body(e.getMessage());
```
여기도 정의된 builder를 사용할 수 있겠네요.